webv2/routes.go

```diff
@@ -55,6 +55,9 @@ func getRoutes() *mux.Router {
 	router.HandleFunc("/SaveRemoteSession", session.SaveRemoteSession).Methods("POST")
 	router.HandleFunc("/ResumeSession/{versionId}", session.ResumeSession).Methods("POST")
 
+	//primarykey
+	router.HandleFunc("/api/primaryKey", primaryKey).Methods("POST")
```

Comment:
/api is not needed as discussed in the call. because we are not following this for other services